inst/rmarkdown/templates/isba/skeleton/LICENSE

```diff
@@ -0,0 +1,415 @@
+The LaTeX Project Public License
```

Comment:
Can we put this LICENCE file in a parent directory ? 

Does it really need to be copied to the user directory when creating a template  ?

Reply:
No I will remove the file.